tests/PerformanceTests/PerformanceTestService/EventSource/BenchmarkMsalDistributedTokenCacheAdapter.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Options;
+using Microsoft.Identity.Web.TokenCacheProviders.Distributed;
+
+namespace PerformanceTestService.EventSource
+{
+    /// <summary>
+    /// Adds benchmarking counters on top of <see cref="MsalDistributedTokenCacheAdapter"/>.
+    /// </summary>
+    public class BenchmarkMsalDistributedTokenCacheAdapter : MsalDistributedTokenCacheAdapter
```

Comment:
Is it used?

Reply:
Yes, it's used in place of normal distributed cache class.